hardhat.config.js

```diff
@@ -0,0 +1,13 @@
+const mainnetFork = {
+  url: 'https://eth-mainnet.alchemyapi.io/v2/lhVWQ3rY2i5_OZtYkU4Lzg_OsDT97Eoz',
+  blockNumber: 13582625,
+}
+
+module.exports = {
+  networks: {
+    hardhat: {
+      chainId: 1,
+      forking: mainnetFork,
+    },
+  },
+}
```

Comment:
not important, but imo we should pass in our own private key value(s) here -- I don't like using the hardhat defaults, just in case someone makes a mistake with them in the future. I'd rather make it explicit that we're using `SUPER_COMPROMISED_PRIVATE_KEY_1` etc.

Reply:
hardhat makes it very explicit that these are compromised when it starts up.